packages/nocodb/src/models/MCPToken.ts

```
@@ -0,0 +1,232 @@
+    const key = `${CacheScope.MCP_TOKEN}:${mcpTokenId}`;
+    await NocoCache.update(key, updateObj);
+
+    return await this.get(context, mcpTokenId, ncMeta);
+  }
+
+  public static async delete(
+    context: NcContext,
+    mcpTokenId: string,
+    ncMeta = Noco.ncMeta,
+  ) {
+    const token = await this.get(context, mcpTokenId, ncMeta);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)